2023-11-21 07:23:04,046 - root - INFO - Initiating present weather
2023-11-21 07:23:04,052 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): radar.bmkg.go.id:8060
2023-11-21 07:23:04,111 - urllib3.connectionpool - DEBUG - https://radar.bmkg.go.id:8060 "GET /getRadarGeotif?token=19387e71e78522ae4172ec0fda640983b8438c9cfa0ca571623cb69d8327&radar=amandemenForecast&type=latest HTTP/1.1" 200 90
2023-11-21 07:23:04,113 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fd4252d7500>
2023-11-21 07:23:04,114 - rasterio.env - DEBUG - Starting outermost env
2023-11-21 07:23:04,114 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-21 07:23:04,114 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fd42562e0b0> created
2023-11-21 07:23:04,117 - rasterio._filepath - DEBUG - Installing FilePath filesystem handler plugin...
2023-11-21 07:23:04,117 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-21 07:23:04,118 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-21 07:23:04,118 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fd42562e0b0>.
2023-11-21 07:23:04,118 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fd4252d7500>
2023-11-21 07:23:04,118 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-21 07:23:14,296 - rasterio._env - INFO - GDAL signalled an error: err_no=11, msg='CURL error: Could not resolve host: radar.bmkg.go.id'
2023-11-21 07:23:14,296 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fd4252d7500>
2023-11-21 07:23:14,296 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fd42562e0b0> options
2023-11-21 07:23:14,296 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fd42562e0b0>.
2023-11-21 07:23:14,296 - rasterio.env - DEBUG - Exiting outermost env
2023-11-21 07:23:14,296 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fd4252d7500>
2023-11-21 07:23:15,881 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fd487e1d2e0>
2023-11-21 07:23:15,881 - rasterio.env - DEBUG - Starting outermost env
2023-11-21 07:23:15,882 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-21 07:23:15,882 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fd466c3aef0> created
2023-11-21 07:23:15,882 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-21 07:23:15,883 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-21 07:23:15,883 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fd466c3aef0>.
2023-11-21 07:23:15,883 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fd487e1d2e0>
2023-11-21 07:23:15,883 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-21 07:23:15,903 - rasterio._base - DEBUG - Nodata success: 0, Nodata value: 0.000000
2023-11-21 07:23:15,904 - rasterio._base - DEBUG - Dataset <open DatasetReader name='/home/swx/present_weather/src/H09_B13_Indonesia_20231121.Z0710.cor.tiff' mode='r'> is started.
2023-11-21 07:23:15,904 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fd487e1d2e0>
2023-11-21 07:23:15,904 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fd466c3aef0> options
2023-11-21 07:23:15,904 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fd466c3aef0>.
2023-11-21 07:23:15,904 - rasterio.env - DEBUG - Exiting outermost env
2023-11-21 07:23:15,905 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fd487e1d2e0>
2023-11-21 07:23:15,905 - rasterio.env - DEBUG - Entering env context: <rasterio.env.Env object at 0x7fd4b28896d0>
2023-11-21 07:23:15,905 - rasterio.env - DEBUG - Starting outermost env
2023-11-21 07:23:15,905 - rasterio.env - DEBUG - No GDAL environment exists
2023-11-21 07:23:15,905 - rasterio.env - DEBUG - New GDAL environment <rasterio._env.GDALEnv object at 0x7fd4255ff610> created
2023-11-21 07:23:15,906 - rasterio._env - DEBUG - GDAL data found in package: path='/home/swx/miniconda3/envs/present_weather/lib/python3.12/site-packages/rasterio/gdal_data'.
2023-11-21 07:23:15,906 - rasterio._env - DEBUG - PROJ_LIB found in environment.
2023-11-21 07:23:15,906 - rasterio._env - DEBUG - Started GDALEnv: self=<rasterio._env.GDALEnv object at 0x7fd4255ff610>.
2023-11-21 07:23:15,906 - rasterio.env - DEBUG - Entered env context: <rasterio.env.Env object at 0x7fd4b28896d0>
2023-11-21 07:23:15,906 - rasterio._base - DEBUG - Sharing flag: 0
2023-11-21 07:23:15,908 - rasterio._base - DEBUG - Nodata success: 0, Nodata value: 0.000000
2023-11-21 07:23:15,908 - rasterio._base - DEBUG - Dataset <open DatasetReader name='/home/swx/present_weather/src/H09_B13_Indonesia_20231121.Z0710.cor.tiff' mode='r'> is started.
2023-11-21 07:23:15,908 - rasterio.env - DEBUG - Exiting env context: <rasterio.env.Env object at 0x7fd4b28896d0>
2023-11-21 07:23:15,908 - rasterio.env - DEBUG - Cleared existing <rasterio._env.GDALEnv object at 0x7fd4255ff610> options
2023-11-21 07:23:15,908 - rasterio._env - DEBUG - Stopped GDALEnv <rasterio._env.GDALEnv object at 0x7fd4255ff610>.
2023-11-21 07:23:15,908 - rasterio.env - DEBUG - Exiting outermost env
2023-11-21 07:23:15,908 - rasterio.env - DEBUG - Exited env context: <rasterio.env.Env object at 0x7fd4b28896d0>
2023-11-21 07:23:15,928 - asyncio - DEBUG - Using selector: EpollSelector
2023-11-21 07:23:19,295 - root - INFO - dask.bag<lambda, npartitions=34>
2023-11-21 07:24:35,288 - root - INFO - Total elapsed time: 0:01:31.243475
